jenkins/diff-job-config-patch.sh

```diff
@@ -56,15 +56,17 @@ readonly common_commands="\
     -o /output/kubernetes-jenkins-pull"
 
 docker run --rm=true -i \
+  -v "${REPO_DIR}:/test-infra:ro" \
   -v "${docker_volume_output_dir}/upstream:/output" \
   "${JOB_BUILDER_IMAGE}" \
-  bash -c "git checkout master && git pull && ${common_commands}"
+  bash -c "cp -r /test-infra /workspace && cd /workspace && \
```

Comment:
`/test-infra` is read-only so a `git checkout master` isn't going to go over well. Plus if you're doing local development you don't want it to overwrite your changes :P